controllers/services/auth/auth_controller.go

```diff
@@ -46,7 +46,6 @@ func NewServiceReconciler(ctx context.Context, mgr ctrl.Manager) error {
 		WithAction(deploy.NewAction(
 			deploy.WithCache(),
 		)).
-		WithAction(setStatus).
 		Build(ctx)
```

Comment:
are we going to have some status on auth in phase 2?

Reply:
I don't know, I've removed the specific action because the basic status handling is done as part of the common reconciliation.

Reply:
i do not get this, which one is the "common reconciliation", `WithAction(deployments.NewAction())`?

Reply:
the `Reconciler` struct now automatically set the `Ready` and `ProvisioningSucceeded` conditions:

https://github.com/opendatahub-io/opendatahub-operator/pull/1684/files#diff-1d07cdba1ef7647d2e1ac39e6f7a861bbb19e44824dafd0f069d0abe96856e4dR282-R304

Reply:
correct